docs/source/references.bib

```diff
@@ -197,6 +197,27 @@ @TechReport{ONeilLaubscher1976
   Url                      = {https://archive.org/details/DTIC_ADA026294}
 }
 
+@techreport{Poder1998,
+  author       = {Knud Poder and Karsten Engsager},
+  year         = {1998},
+  title        = {Some Conformal Mappings and Transformations for Geodesy
+                  and Topographic Cartography},
+  address      = {Copenhagen, Denmark},
+  institution  = {National Survey and Cadastre},
+  type         = {National Survey and Cadastre Publications},
+  series       = {4},
+  volume       = {6},
+  isbn         = {87-7866-085-8},
+  pages        = {63}
+}
+
+@Misc{Rittri2012,
+    author = {Mikael Rittri},
+    title  = {New omerc approximations of Denmark System 34},
+    year   = {2012},
+    url    = {http://lists.maptools.org/pipermail/proj/2012-June/006323.html}
```

Comment:
can you change this to https://lists.osgeo.org/pipermail/proj/2012-June/005926.html  , since maptools.org is probably going to be retired at some point

Reply:
Thanks for noticing this - I had planned to change it, but forgot about it